environs/config/config.go

```diff
@@ -825,6 +828,11 @@ func (c *Config) UUID() string {
 	return c.mustString(UUIDKey)
 }
 
+// DefaultSpace returns the default-space for the model.
+func (c *Config) DefaultSpace() string {
+	return c.mustString(DefaultSpace)
```

Comment:
@manadart As I am not sure what application and boundaries this key has I will make sure that it is a string.

Reply:
String is correct.